From e583f5c3f3207661f23956283f7a5117c3226a1b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Tue, 12 Sep 2017 10:31:11 -0400
Subject: [PATCH 1/2] Set release configuration for cartographer.

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index aa010fa..4ca310e 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -26,7 +26,7 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure -- -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)"
+	dh_auto_configure @(debhelper_autoconfigure_options) -DCMAKE_BUILD_TYPE=Release
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.7.4

